contracts/calypso/controller/controller.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"go.dedis.ch/dela/cli"
+	"go.dedis.ch/dela/cli/node"
+	"go.dedis.ch/dela/contracts/calypso"
+	"go.dedis.ch/dela/core/access"
+	"go.dedis.ch/dela/core/execution/native"
+	"golang.org/x/xerrors"
+)
+
+// aKey is the access key used for the calypso contract
+var aKey = [32]byte{3}
+
+// miniController is a CLI initializer to register the value contract
+//
+// - implements node.Initializer
+type miniController struct {
+}
+
+// NewController creates a new minimal controller for the value contract.
+func NewController() node.Initializer {
```

Comment:
why node.Initializer and not calypso.Initializer ? I have to look in dela to understand where this comes from.